apps/fz_http/lib/fz_http/devices/device.ex

```diff
@@ -211,6 +213,26 @@ defmodule FzHttp.Devices.Device do
     end
   end
 
+  defp validate_single_ip(%Ecto.Changeset{changes: %{ipv4: %{netmask: 32}}} = changeset, :ipv4),
```

Comment:
Prob want to disallow specifying the mask because I think we append /32 and /128 currently

Reply:
Then you can just check for nil I think to pass validation, then error if the netmask exists

Reply:
I tested it and it works correctly with /32 or /128 no double slash. Maybe postgres inet deletes the slash? Will erase it so we don't depend on an undocumented feature.

Reply:
It seems that the /32 or /128 are automatically added when the changeset is created(but not when migrations run) so I need to keep both the test for /32 and `nil`